src/player.cpp

```diff
@@ -825,12 +827,37 @@ static void OnMapSaveFileReady(FileRequestResult*) {
 }
 
 void Player::LoadSavegame(const std::string& save_name) {
+	Output::Debug("Loading Save %s", FileFinder::GetPathInsidePath(Main_Data::GetSavePath(), save_name).c_str());
+
 	std::unique_ptr<RPG::Save> save = LSD_Reader::Load(save_name, encoding);
 
 	if (!save.get()) {
 		Output::Error("%s", LcfReader::GetError().c_str());
 	}
 
+	std::stringstream verstr;
+	int32_t ver = save->easyrpg_data.version;
+	if (ver == 0) {
+		verstr << "RPG_RT or EasyRPG Player Pre-0.6.0";
+	} else if (ver >= 10000) {
+		verstr << "Unsupported";
+	} else {
+		verstr << "EasyRPG Player ";
+		char verbuf[6];
+		sprintf(verbuf, "%d.%d.%d", ver / 1000 % 10, ver / 100 % 10, ver / 10 % 10);
```

Comment:
Use `snprintf()` for extra safety

Reply:
I wanted to but I somehow expect that at least one platform port has a broken stdlib, well I will change it an dwe will see what happens.

Reply:
Also I would just make the buffer large like 64 bytes. What if we have `10.0.0` or something bigger one day? Are you going to remember to resize the buffer? I know I wouldn't.

Reply:
I see, if you make it bigger (like 32 or more), `sprintf()` is probably fine.